toolbox/dicom.py

```diff
@@ -170,10 +170,35 @@ def load_scan(dicom_dir_path):
     slices.sort(key=lambda s: s.SliceLocation)
     return slices
 
+class DicomReader:
+    """Read a dicom series by SimpleITK"""
+    def __init__(self, dcm_root: str):
+        """ Init the class by a given dcm_root.
 
-def read_from_dicom_files(dicom_file_paths):
-    reader = sitk.ImageSeriesReader()
-    dicom_names = reader.GetGDCMSeriesFileNames(dicom_file_paths)
-    reader.SetFileNames(dicom_names)
-    itk_image = reader.Execute()
-    return itk_image
\ No newline at end of file
+        :param dcm_root: a folder path, it should be the parent folder of a dicom series,
+        for example /kaggle/input/rsna-miccai-brain-tumor-radiogenomic-classification/train/00000/FLAIR/.
+        """
+        self.dcm_root = dcm_root
+        self.itk_image = self.read_from_dicom_series()
+
+    def get_array(self):
+        return sitk.GetArrayFromImage(self.itk_image)
```

Comment:
why doing this ha ha. It's not that valuable, to be honest, but we are not building sdintegration, so totally fine!

Reply:
I saw how it's used below. It looks fine!